tools/cmake/mbed_greentea.cmake

```diff
@@ -70,6 +81,17 @@ macro(mbed_greentea_add_test)
 
     mbed_set_post_build(${MBED_GREENTEA_TEST_NAME})
 
+    add_test(
+        NAME ${MBED_GREENTEA_TEST_NAME}
+        COMMAND mbedhtrun
+            -m ${MBED_TARGET}
+            -f ${CMAKE_CURRENT_BINARY_DIR}/${MBED_GREENTEA_TEST_NAME}.bin
+            -p ${MBED_TARGET_SERIAL_PORT}
+            -t ${MBED_TARGET_SERIAL_NUM}
+            -d ${MBED_TARGET_MOUNT_POINT}
+            -e ${MBED_GREENTEA_HOST_TESTS_DIR}
```

Comment:
Is it possible to make it optional? For example, many tests do not require any interaction with the host apart from the "start" signal and reporting of success/failure, in which case `default_auto.py` from `htrun` itself is used.